File: MODUL-2/SOAL 4/PRAK204-2410817120017-RIKA-FAULIANA-RAHMI.py
def hitung_tabung(r, t):
    pi = 22 / 7
    volume = pi * r**2 * t
    luas = 2 * pi * r * (r + t)
    keliling = 2 * pi * r
    
    # Format hasil dengan 2 angka di belakang koma
    print(f"Volume = {volume:.2f}")
    print(f"Luas = {luas:.2f}")
    print(f"Keliling = {keliling:.2f}")

# Input dari pengguna
r = float(input("Masukkan jari-jari: "))
t = float(input("Masukkan tinggi: "))

# Hitung dan tampilkan hasil
hitung_tabung(r, t)

File: MODUL-2/SOAL 4/PRAK204-2410817120017-RIKA-FAULIANA-RAHMI.c
#include <stdio.h>

#define PI 22.0 / 7.0  // Menggunakan pendekatan pi sebagai 22/7

void hitung_tabung(float r, float t) {
    float volume = PI * r * r * t;
    float luas = 2 * PI * r * (r + t);
    float keliling = 2 * PI * r;

    // Tampilkan hasil dengan 2 angka di belakang koma
    printf("Volume = %.2f\n", volume);
    printf("Luas = %.2f\n", luas);
    printf("Keliling = %.2f\n", keliling);
}

int main() {
    float r, t;

    printf("Masukkan jari-jari: ");
    scanf("%f", &r);
    printf("Masukkan tinggi: ");
    scanf("%f", &t);

    // Hitung dan tampilkan hasil
    hitung_tabung(r, t);

    return 0;
}
